src/plugins/core/sheet.ts

```diff
@@ -834,10 +834,10 @@ export class SheetPlugin extends CorePlugin<SheetState> implements SheetState {
   }
 
   private moveCellOnColumnsDeletion(sheet: Sheet, deletedColumn: number) {
-    for (let [index, row] of Object.entries(sheet.rows)) {
-      const rowIndex = parseInt(index, 10);
```

Comment:
> This saves 20-40ms on a 10.000 rows sheet with 26 columns, when deleting the
first column.

to you have an rough idea of the total time before/after?

Reply:
I don't have the performance snapshot anymore, but it doesn't really save that much time (20-60ms over 500-700ms I think). But hey it's free.